firebase/test/core/data/repositories/memos-repository.spec.ts

```diff
@@ -0,0 +1,128 @@
+import * as assert from "assert";
+import * as sinon from "sinon";
+import { FirestoreGateway } from "#data/gateways/firestore-gateway";
+import { MemosRepository } from "#data/repositories/memos-repository";
+import Ajv2020 from "ajv/dist/2020";
+import { SchemaValidator } from "#data/schemas/schema-validator";
+import { Memo } from "#domain/models/memo";
+import SerializationError from "#faults/errors/serialization-error";
+
+describe("MemosRepository", () => {
+  const firestoreMock = sinon.createStubInstance(FirestoreGateway);
+  const schemaMock = new SchemaValidator(new Ajv2020());
+  const memosRepo = new MemosRepository(<FirestoreGateway>(<any>firestoreMock), schemaMock);
```

Comment:
I had to perform this type manipulation to TS don't complaint about the `sinon.Stub` type. I had absolutely no idea why it's complaining and I lost a lot of time trying to figure it out. If you have any idea about what's possible happening here, I'm open to suggestions.

Reply:
Awesome